data_tools/add.py

```diff
@@ -27,11 +27,11 @@
 
 
 if CONTENT_TYPE not in ["users", "schemes", "messages"]:
-    print ("update target {} not known".format(TARGET))
+    print ("update target {} not known".format(CONTENT_TYPE))
```

Comment:
I think it should be "update content type {} not known"

Reply:
Thanks, adjusted